ghcup-0.0.7.yaml

```diff
@@ -3610,6 +3609,101 @@ ghcupDownloads:
               dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-aarch64-apple-darwin.tar.xz
               dlSubdir: ghc-9.4.7-aarch64-apple-darwin
               dlHash: 5d85f9836d72d45634039218ed52e9faa0ed00c0db056f3d1162b4c2b3838e38
+    9.4.8:
+      viTags:
+      - base-4.17.2.1
+      - Recommended
```

Comment:
We only set to `Recommended`  if there's a HLS release already for this version. Is that the case?

Reply:
Not yet, we need the ghcup metadata updated before we can do an HLS release. I will remove it and add it with the HLS metadata